### ISSUE TITLE
An Error for a metal free compound

### ISSUE THREAD

**Issue by ichxw** [PodewitzLab/PyConSolv#29]
An error occurred after trying the code on a metal free compound. Here is the error: 
```
Copying example.xyz to input.xyz

Traceback (most recent call last):
  File "/home/xzhu/anaconda3/envs/PyConSolv/bin/pyconsolv", line 8, in <module>
    sys.exit(main())
  File "/home/xzhu/anaconda3/envs/PyConSolv/lib/python3.10/site-packages/PyConSolv/pyconsolv.py", line 51, in main
    conf = PyConSolv(inputfilepath)
  File "/home/xzhu/anaconda3/envs/PyConSolv/lib/python3.10/site-packages/PyConSolv/ConfGen.py", line 132, in __init__
    self.metalCheck()
  File "/home/xzhu/anaconda3/envs/PyConSolv/lib/python3.10/site-packages/PyConSolv/ConfGen.py", line 713, in metalCheck
    if line.split()[0].upper() in self.metals:
IndexError: list index out of range

```
Looks like it was trying to find a metal atom in the xyz file but encountered an empty line(the second line). The xyz file was generated in  Avogadro. 
Here is my xyz file: 
```
25

C          1.20370        0.58950       -0.03830
C          0.04240        1.35570        0.04730
H          0.11810        2.40960        0.11280
C         -1.19900        0.72920        0.04280
H         -2.08400        1.30890        0.10550
C         -1.28210       -0.65960       -0.04410
H         -2.22980       -1.13430       -0.04700
C         -0.11880       -1.42360       -0.12620
H         -0.18550       -2.47950       -0.19000
C          1.12680       -0.80450       -0.12320
H          2.00950       -1.38410       -0.17980
N          5.29260        1.79350        0.10670
H          6.25520        2.08200        0.17140
C          4.58060        1.84530       -1.17180
H          5.24320        1.52350       -1.97650
H          4.29190        2.88040       -1.36250
C          3.30180        0.98200       -1.19010
H          3.58590       -0.06180       -1.20860
H          2.75810        1.16020       -2.10910
N          2.43360        1.25650       -0.02990
C          3.20300        1.08130        1.25530
H          3.16910        0.04600        1.56430
H          2.69840        1.61040        2.05160
C          4.68170        1.50890        1.25970
O          5.34310        1.57690        2.28730
```

**Comment by Lapsis-glitch** [comment]
Hi! I will look into it. Thanks!

**Comment by Lapsis-glitch** [comment]
Fixed with 814cd2f281af825228357a183f787dc58ca73952
